### DISCUSSION TITLE
Release roadmap. Any ETA for 10.8.0?

### DISCUSSION THREAD

**Discussion by CluelessTechnologist** [jellyfin/jellyfin#6717] [General] [answered]
Hey,
I have an [issue ](https://github.com/jellyfin/jellyfin/issues/4991)that has been resolved in 10.8.0 so I'm interested to know when the next major release of Jellyfin is getting released.
Is there a roadmap somewhere?

Also does fixes in 10.8.0 trickle down to 10.7.x releases?

**Comment by nielsvanvelzen** [comment, marked as answer]
> Any ETA for 10.8.0?

No

> Is there a roadmap somewhere?

no, only the issue/pr list

> Also does fixes in 10.8.0 trickle down to 10.7.x releases?

10.7.7 should be the last release for the 10.7 cycle, so no

**Reply by CluelessTechnologist** [reply]
Thanks, since 10.7.7 is the final release of 10.7 it shouldn't be too long of a wait 😄
